source/shared/core_util.cpp

```diff
@@ -414,3 +414,189 @@ unsigned int convert_string_from_default_encoding( _In_ unsigned int php_encodin
 }
 
 }
+
+
+namespace data_classification {
+    const char* DATA_CLASS = "Data Classification";
+    const char* LABEL = "Label";
+    const char* INFOTYPE = "Information Type";
+    const char* NAME = "name";
+    const char* ID = "id";
+
+    void convert_sensivity_field(_Inout_ sqlsrv_stmt* stmt, _In_ SQLSRV_ENCODING encoding, _In_ unsigned char *ptr, _In_ int len, _Inout_updates_bytes_(cchOutLen) char** field_name)
```

Comment:
I did that because I need the encoding when doing the parsing, and rather than check the following every single time, I did it one time in the beginning: 
`SQLSRV_ENCODING encoding = ((stmt->encoding() == SQLSRV_ENCODING_DEFAULT ) ? stmt->conn->encoding() : stmt->encoding());`